libx509pq.c

```diff
@@ -638,6 +639,130 @@ Datum x509_publickeymd5(
 	PG_RETURN_NULL();
 }
 
+PG_FUNCTION_INFO_V1(x509_tbscert_strip_sct);
```

Comment:
I'm also thinking ahead to 6962-bis, which doesn't have the poison extension and has a different OID for the SCTs extension.